wandb/util.py

```diff
@@ -881,6 +881,23 @@ def no_retry_4xx(e: Exception) -> bool:
     raise UsageError(body["errors"][0]["message"])
 
 
+def parse_backend_error_messages(response: requests.Response) -> List[str]:
+    errors: List[str] = []
+    try:
+        data = response.json()
+    except ValueError:
+        return errors
+
+    if "errors" in data and isinstance(data["errors"], list):
+        for error in data["errors"]:
+            # Our tests and potentially some api endpoints return a string error?
+            if isinstance(error, str):
+                error = {"message": error}
+            if "message" in error:
+                errors.append(error["message"])
+    return errors
```

Comment:
suggestion(optional): as usual for existing code, please consider this is optional -- suggestion is mainly intended to improve readability of an existing (+very negligibly improve perf):

```suggestion
    try:
        data = response.json()
        error_objs = data["errors"]
    except ValueError:
        return [] # From decoding JSON response
    except (LookupError, TypeError): 
        return [] # From getting invalid JSON key/path

    error_msgs: List[str] = []
    if isinstance(error_objs, list):
        for error in error_objs:
            # Our tests and potentially some api endpoints return a string error?
            if isinstance(error, str):
                error_msgs.append(error)
            else:
                with suppress(LookupError, TypeError):
                    error_msgs.append(error["message"])
    return errors
```